src/main/java/org/spongepowered/api/world/explosion/Explosion.java

```diff
@@ -91,6 +91,32 @@ static Builder builder() {
      */
     boolean shouldDamageEntities();
 
+    /**
+     * Gets the number of rays cast per length of each side of the explosion bounding box.
+     *
+     * @return The number of rays cast per length of each side of the explosion bounding box.
+     */
+    int getResolution();
```

Comment:
Because not default implementing this leads to a binary breakage for any thing that hasn't implemented the interface in previous builds, this would need to be default implemented to return the default resolution. This will allow for classes that implemented the class prior to this change to still be usable in other environments.